README.md

```diff
@@ -102,15 +102,15 @@ For example, to submit the test jobs in `config/default_jobs.yaml` via
 `--json-string`, run the following command:
 
 ```bash
-docker exec spark-node-master ./submit.sh \
+docker exec spark-node-master-prod ./submit.sh \
     --json-string "$(yq -o=json .test_jobs ./config/default_jobs.yaml)"
```

Comment:
Cron jobs calling Spark will now need to specify a `-dev` or `-prod` suffix to target the new container names.